pkg/adapter/mtping/adapter.go

```diff
@@ -126,5 +126,10 @@ func (a *mtpingAdapter) Remove(ctx context.Context, source *v1beta2.PingSource)
 		a.entryidMu.Lock()
 		delete(a.entryids, key)
 		a.entryidMu.Unlock()
+
+		logger := logging.FromContext(ctx)
+		logger.Infow("Removing pingsource", "namespace", source.Namespace, "name", source.Name,
+			"tenantNamespace", source.Namespace) // Used to route message to tenant
```

Comment:
I find using fields easier to read (and more efficient), and less risk to miss a value. Example: `zap.String("namespace", source.Namespace)`.

Reply:
Really that looks awkward to me. Most examples I've see of infow use that.  
https://blog.sandipb.net/2018/05/04/using-zap-working-with-global-loggers/
If we are going to use that we might as well use desugar thats the fastest. 

// Errorw logs a message with some additional context. The variadic key-value
// pairs are treated as they are in 
key value pairs